pkg/engine/images.go

```diff
@@ -11,14 +11,45 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
+func createImageReferenceFragment(agentPoolProfileName string, windowsProfile *api.WindowsProfile) *compute.ImageReference {
```

Comment:
nit: if this function is specific to Windows images, maybe the name should reflect that

Reply:
updated